README.md

```diff
@@ -73,6 +73,16 @@ are available. These can be set through the environment variable
 OTEL_RESOURCE_ATTRIBUTES="service.version=1.2.3,deployment.environment=production"
 ```
 
+alternatively, if needed, more attributes can be added in code using:
```

Comment:
Won't the code below replace the resource, not add to it?

Reply:
`c.resource=` actually merges, not overwrites, <https://github.com/open-telemetry/opentelemetry-ruby/blob/main/sdk/lib/opentelemetry/sdk/configurator.rb#L69> but something is broken in tests looks like